ldapauthenticator/tests/conftest.py

```diff
@@ -28,6 +28,10 @@ def authenticator():
     authenticator.escape_userdn = True
     authenticator.attributes = ["uid", "cn", "mail", "ou"]
     authenticator.use_lookup_dn_username = False
+    authenticator.group_filter = (
+        "(|(member={userdn})(uniqueMember={userdn})(memberUid={uid}))"
+    )
+    authenticator.group_attributes = ["member", "uniqueMember", "memberUid"]
```

Comment:
This are the default values, do you need to set it?

Reply:
No, no special reason for setting them. I think I built the configuration during tests and didn't consider what to do about configuration default values.

I have removed the values from the test. Also checked the other values, and I think these two were the only two default values.

Thanks @manics !